aio/tools/transforms/angular-base-package/processors/generateKeywords.js

```diff
@@ -120,63 +114,64 @@ module.exports = function generateKeywordsProcessor(log, readFilesProcessor) {
   };
 };
 
-
 function isString(value) {
   return typeof value == 'string';
 }
 
-function convertToMap(collection) {
-  const obj = {};
-  collection.forEach(key => { obj[key] = true; });
-  return obj;
-}
-
-// If the heading contains a name starting with ng, e.g. "ngController", then add the
-// name without the ng to the text, e.g. "controller".
-function tokenize(text) {
-  const rawTokens = text.split(/[\s\/]+/mg);
+function tokenize(text, ignoreWords, dictionary) {
+  // Split on whitespace and things that are likely to be HTML tags (this is not exhaustive but reduces the unwanted tokens that are indexed).
+  const rawTokens = text.split(/[\s\/]+|<\/?[a-z]+(?:\s+\w+(?:="[^"]+")?)*>/img);
```

Comment:
Nits:
- `/` does not need escaping inside a `[...]`.
- `[a-z]` and `\w` do not match `-`, which often appears in HTML tag/attribute names (esp. with custom elements).
- You probably don't need the `multiline` flag :smiley:

```suggestion
  const rawTokens = text.split(/[\s/]+|<\/?[\w\-]+(?:\s+[\w\-]+(?:="[^"]+")?)*>/mg);
```

Reply:
I specifically did not want to capture custom-elements since those are often things that we want to search on.
Primarily this was to avoid things like `table`, `tr`, `td`, `br`, `div` appearing in the index.

Reply:
What would we want to match in a custom element's opening/closing tags? 😕

Reply:
https://angular.io/?search=app-hero-detail

Reply:
Fair enough :smiley: